sensor_msgs/Image shoulderImage
sensor_msgs/Image gripperImage
